Gemfile.lock

```diff
@@ -32,11 +32,11 @@ GEM
       erubis (~> 2.7.0)
       rails-dom-testing (~> 1.0, >= 1.0.5)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
-    active_model_serializers (0.10.4)
+    active_model_serializers (0.10.6)
```

Comment:
A bunch of specs broken probably because of: https://github.com/rails-api/active_model_serializers/blob/0-10-stable/CHANGELOG.md